### ISSUE TITLE
HTTPError: 401 Client Error: Unauthorized for url: https://api.ring.com/clients_api/session

### ISSUE THREAD

**Issue by drummer0417** [tchellomello/python-ring-doorbell#150]
******** Sorry!! ******** i guess this issue was already raised ... 
consider it not posted. Can be removed.
.

Hi,

Since yesterday evening i get the following error while running this line:
myring = Ring(ringUsername,ringPassword)

Error: requests.exceptions.HTTPError: 401 Client Error: Unauthorized for url: https://api.ring.com/clients_api/session

I am running your latest branche (updated two days ago and was running fine until last night)

Regards,
Hans

**Comment by asantaga** [comment]
Dupe of https://github.com/tchellomello/python-ring-doorbell/issues/146

Workaround
Modify __init__.py method to include

```
 from ring_doorbell.const import (HEADERS)
HEADERS["User-Agent"] = "ring/5.21.0.5 CFNetwork/1121.2.2 Darwin/19.2.0"
```

**Comment by drummer0417** [comment]
thanks!

Groet,

Hans

Op wo 25 dec. 2019 13:39 schreef Angelo Santagata <notifications@github.com

**Comment by tchellomello** [comment]
Closed dup #146
